tests/test_stream.py

```diff
@@ -0,0 +1,61 @@
+from queue import Queue
+from morphocut import Pipeline
+from morphocut.stream import Slice, StreamBuffer, PrintObjects
+
+import pytest
+
+
+def test_Slice():
+    # Assert that the stream is sliced
+    items = "ABCDEFG"
+
+    with Pipeline() as pipeline:
+        result = Slice(2)
+
+    obj = result.transform_stream(items)
```

Comment:
You do not use the pipeline at all here. Use `stream = pipeline.transform_stream(items)` (like I said before). Then transform the stream to a list. Then assert that the list contains the right items.